build.gradle

```diff
@@ -17,7 +17,7 @@ plugins {
     id "info.solidsoft.pitest" version '1.7.0'
     id 'org.owasp.dependencycheck' version '6.2.2'
     id 'au.com.dius.pact' version '4.2.14'
-    id 'com.github.hmcts.rse-cft-lib' version '0.19.380'
+    id 'com.github.hmcts.rse-cft-lib' version '0.19.380+'
```

Comment:
Think this should be '0.19.+' . Also, not for now, but I think the new preferred standard is ivy notation (though I may be out of date)